test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
we really need to figure out why these boms started appearing...editorconfig changes mby?